src/components/welcome/WelcomeSection.vue

```diff
@@ -0,0 +1,130 @@
+<script>
+import { defineComponent } from "vue";
+import SectionContainer from "@/components/global/SectionContainer.vue";
+import SectionTitlePrimary from "@/components/global/SectionTitlePrimary.vue";
+import SectionDescription from "@/components/global/SectionDescription.vue";
+import BaseButton from "@/components/global/BaseButton.vue";
+import BaseImage from "@/components/global/BaseImage.vue";
+
+export default defineComponent({
+  name: "WelcomeSection",
+  components: {
+    SectionContainer,
+    SectionTitlePrimary,
+    SectionDescription,
+    BaseButton,
+    BaseImage,
+  },
+  data() {
+    return {
+      userLogged: false,
+    };
+  },
+  computed: {
+    routeName() {
+      return this.userLogged ? { name: "BookTicket" } : { name: "UserLogIn" };
+    },
```

Comment:
I know it's dealing with temporary code with the local state but still, I think it would be better named as `bookingRoute` not route**Name** because, well it's not only a name, it's a router location definition 😄

If you want to keep it as name then you could return just the string and use it appropriately in the template as:

`:to="{ name: routeName }"`